apps/next-evans/package.json

```diff
@@ -9,8 +9,8 @@
     "typecheck": "tsc --noEmit",
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
-    "format": "just ../../prettier",
-    "format:check": "just ../../prettier-check"
+    "format": "just prettier",
+    "format:check": "just prettier-check"
```

Comment:
these two lines look like a rebase mistake, they should be the other way, i think